modules/openapi-generator/src/main/resources/csharp-netcore/appveyor.mustache

```diff
@@ -0,0 +1,13 @@
+# auto-generated by OpenAPI Generator (https://github.com/OpenAPITools/openapi-generator)
+#
+image: Visual Studio 2019
+clone_depth: 1
+before_build:
+- dotnet restore
+build:
+  project: {{{packageName}}}.sln
+  verbosity: minimal
+test_script:
+- dotnet test -c Release .\src\{{{packageName}}}.Test\{{{packageName}}}.Test.csproj
```

Comment:
```suggestion
- dotnet test -c Release
```

dotnet test automatically searches for test projects included in the solution, so there is no need to explicitly specify projects.